src/Control/Email/Mailer.php

```diff
@@ -2,10 +2,10 @@
 
 namespace SilverStripe\Control\Email;
 
-use InvalidArgumentException;
-use SilverStripe\Control\HTTP;
-use SilverStripe\Core\Convert;
+use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Core\Object;
+use Swift_Mailer;
+use Swift_MailTransport;
```

Comment:
So I've taken a bit of a look at how Laravel does it and it's quite closely coupled - mostly because devs can write their own transports (which is basically the bulk of what they'd probably need to do).

There are a lot of out of the box transports, too, which would likely cover most of what people need.